static/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -5,3 +5,14 @@
     {{/tr}}
 </p>
 <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
+<p>
+    {{#tr}}
+        <strong>{username}</strong> has <z-invites></z-invites>, and {bot_text}
+        {{#*inline "z-invites"}}<span class="number_of_invites_by_user">0 unexpired invitations</span>{{/inline}}
```

Comment:
This i18n approach doesn't work -- the `0 unexpired invitations` part cannot be translated.